packages/fast-components/src/card/card.template.ts

```diff
@@ -0,0 +1,8 @@
+import { html } from "@microsoft/fast-element";
+import { Card } from "./card";
+
+export const CardTemplate = html<Card>`
+  <div part="root">
+    <slot></slot>
+  </div>
```

Comment:
@nicholasrice depending on where we land with the conversation on your PR, do you think we even need a root class on the card here? Would it be unintuitive to just remove it and leave it off? Ultimately you just target the host or the stuff you put inside the card. It may be an outlier from our other components in terms of consistency, but it seems logical to me to remove it...

Reply:
I agree, it's probably not needed in this case.

Reply:
checkbox has no "root" element. I actually think *most* components (if any) will need a "root" element, since the host essentially serves that function.

Reply:
updated this